ast.hpp

```diff
@@ -1667,9 +1667,6 @@ namespace Sass {
         if (has_rest_parameter_) {
           error("functions and mixins cannot have more than one variable-length parameter", p->pstate());
         }
-        if (has_optional_parameters_) {
```

Comment:
Is this correct? I believe this is still a valid error.\

The error case mentioned in the issue is 

> [when] optional parameter followed by a variable-length parameter

This change appears to affect the case 

> [when] a variable-length parameter is followed an optional parameter

Which is a legal scenario.

If this fixes the reported issues I believe the bug is due to something before this check.